bitcoind-rpc/src/main/scala/org/bitcoins/rpc/serializers/JsonSerializers.scala

```diff
@@ -354,9 +357,54 @@ object JsonSerializers {
 
   implicit val rpcPsbtInputReads: Reads[RpcPsbtInput] = RpcPsbtInputReads
 
-  implicit val decodePsbtResult: Reads[DecodePsbtResult] =
+  implicit val decodePsbtResultReads: Reads[DecodePsbtResult] =
     Json.reads[DecodePsbtResult]
 
+  implicit val psbtMissingDataReads: Reads[PsbtMissingData] =
+    Json.reads[PsbtMissingData]
+
+  implicit val analyzePsbtInputReads: Reads[AnalyzePsbtInput] =
+    Json.reads[AnalyzePsbtInput]
+
+  implicit val analyzePsbtResultReads: Reads[AnalyzePsbtResult] =
+    Json.reads[AnalyzePsbtResult]
+
+  implicit val getNodeAddressesReads: Reads[GetNodeAddressesResult] = { js =>
+    for {
+      time <- js.validate[Long].map(FiniteDuration(_, TimeUnit.SECONDS))
```

Comment:
Instead of using `TimeUnit` in a bunch of places in this file, you should import `scala.concurrent.duration.DurationLong` which will let you call a bunch of new methods on any long such as `_.seconds` and `_.microseconds` that return `FiniteDuration`s